core/src/main/java/bio/terra/pearl/core/dao/BaseJdbiDao.java

```diff
@@ -280,6 +280,21 @@ protected List<T> findAllByTwoProperties(String column1Name, Object column1Value
         );
     }
 
+    protected List<T> findAllByTwoProperties(String column1Name, Object column1Value,
+                                             String column2Name, Collection<?> column2Values) {
```

Comment:
Should both objects be collections to make them consistent? I see what you mean about needing more composable methods... Maybe a generic `findByAllProperties` that works like creating a map; `findByAllProperties('prop1', List.of('val1', 'val2'), 'prop2', List.of('val3', 'val4'), ...)`

Reply:
there already is a findAllTwoProperties that takes two collections--see immediately below this.  The idea is that you call findAllByTwoProperties with what you have, and then based on whether it's a collection or not, the right method will get invoked to construct the right query

Reply:
Oh, I understand. That makes more sense in context.